cudamapper/src/overlapper_naive.cpp

```diff
@@ -102,13 +82,12 @@ namespace claragenomics {
 
         std::vector<Overlap> overlaps;
 
-        for( std::map<std::pair<int,int>, Overlap>::iterator it = reads_to_overlaps.begin(); it != reads_to_overlaps.end(); ++it ) {
-            auto overlap = it->second;
+        for(const auto& it: reads_to_overlaps) {
+            const auto& overlap = it.second;
             if (overlap.overlap_complete){
-                overlaps.push_back(it->second);
+                overlaps.push_back(it.second);
```

Comment:
Why not use `overlap` directly? (I don't think it would have an effect on performance, but makes it more readable)

Reply:
agreed